packages/wdio-applitools-service/src/index.js

```diff
@@ -50,6 +50,16 @@ export default class ApplitoolsService {
 
             return this.eyes.check(title, Target.window())
         })
+
+        global.browser.addCommand('takeRegionSnapshot', (title, region) => {
+            if (!title) {
+                throw new Error('A title for the Applitools snapshot is missing')
+            } else if (!region || region === null) {
+                throw new Error('A region for the Applitools snapshot is missing')
+            }
+
+            return this.eyes.check(title, region)
```

Comment:
How about:

```suggestion
            return this.eyes.check(title, Target.region(region))
```

So that people don't need to also know the Applitools interfaces so they can just do:

```js
browser.takeRegionSnapshot('Reddit icon; main page', 'css=a._30BbATRhFv3V83DHNDjJAO')
```

or maybe even just

```js
browser.takeRegionSnapshot('Reddit icon; main page', 'a._30BbATRhFv3V83DHNDjJAO')
```

Reply:
This is updated.